ui/app/templates/components/transit-key-action/decrypt.hbs

```diff
@@ -33,7 +44,18 @@
       <div class="field">
         <label for="ciphertext" class="is-label">Ciphertext</label>
         <div class="control">
-          {{textarea id="ciphertext" name="ciphertext" value=ciphertext class="textarea" data-test-transit-input="ciphertext"}}
+          {{ivy-codemirror
+            value=ciphertext
+            valueUpdated=(action (mut ciphertext))
+            options=(hash
```

Comment:
does this need the `id="ciphertext"` as well?

Reply:
Turns out the id's aren't passed along to the Codemirror component :/ so I eliminated it to avoid confusion, and instead wrap each of the ivy-codemirror components with a unique ID.

If we see a reason to put back the ID I'm happy to do so!

Reply:
oh that's excellent, nice thinking! thank you for removing dead code!